src/test/resources/kafka-integration-no-namespace.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would be better, if you'll move this file to the appropriate  package and in the `test` dir, not `resources`. It easier to navigate from classes to their contexts and vise versa.